### ISSUE TITLE
Not smooth

### ISSUE THREAD

**Issue by DeadLyBro** [davidparsson/scrollbar-anywhere#77]
My setup:
- Mouse button used when scrolling: CTRL + Left Click
- Chrome version: 114.0.5735.2469 (64 bit) [ its actually Yandex 23.7.0.2469 (64-bit) ]
- Operating system: Windows 10 22H2 ( 19045.3208 ) 

My issue; When i drag and release the keys, it look laggy, like frame drop in games. So it's not smooth. Please fix if you can thanks.

**Comment by davidparsson** [comment]
Do you see this on every web page or a few larger ones?

**Comment by DeadLyBro** [comment]
Larger ones i think. I'm reading webtoon/manga. Normally there is no lag but if i use this extension to drag down the page its laggy like drop in games.

**Comment by DeadLyBro** [comment]
I hope you can understand.

**Comment by DeadLyBro** [comment]
I tried ScrollEverywhere ( I hope there is no punishment for describe other extension names. ) that one is good scrolling but this extension is laggy when i release the keys.

**Comment by davidparsson** [comment]
Ok. Does what you see correspond to what is already reported in #48?

**Comment by DeadLyBro** [comment]
Yes, as you mentioned in this https://github.com/davidparsson/scrollbar-anywhere/issues/48#issuecomment-197866767 , something like stuttering is happening.

**Comment by davidparsson** [comment]
I’m not at all sure. I’m sorry to say that the time I’ve got to spend on projects like this is close to none these days, and this project has pretty low priority for me. If I recall correctly I’ve had a hard time producing this as well.

Pull requests are welcome for anyone who’s able.

Since this seems to describe the same issue as in #48 I’m going to close this. Feel free to reopen if you don’t agree.
